### PR TITLE
Integrate Mycelium with tfcmd

### PR DESCRIPTION
**Pull request by AbdelrahmanElawady**
### Description

Integrate Mycelium with tfcmd

### Changes

- VMs and K8s can be deployed with Mycelium enable by passing `--mycelium` and `--workers-mycelium`.

### Related Issues

- #907 

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstring

### CODE REVIEW

File: grid-cli/cmd/deploy_kubernetes.go [hunk 1]
```diff
@@ -65,15 +65,28 @@ var deployKubernetesCmd = &cobra.Command{
...
+
+		mycelium, err := cmd.Flags().GetBool("mycelium")
+		if err != nil {
+			return err
+		}
+		var seed []byte
+		if mycelium {
+			seed, err = workloads.RandomMyceliumIPSeed()
+			if err != nil {
+				log.Fatal().Err(err).Send()
```
Comment (rawdaGastan) [inline grid-cli/cmd/deploy_kubernetes.go:77]: Why you log `fatal` here?

**Review by rawdaGastan — CHANGES_REQUESTED** [review]
can we add mycelium flags for vm and kubernetes docs :D

File: grid-cli/cmd/deploy_kubernetes.go (hunk 1 above)
Comment (AbdelrahmanElawady) [reply grid-cli/cmd/deploy_kubernetes.go:77]: as we can't proceed with vm deployment if we failed to generate a seed, so we exit the process

File: grid-cli/docs/vm.md [hunk 2]
```diff
@@ -59,57 +65,52 @@ Example:
...
         "SolutionProvider": null,
         "NetworkName": "examplevmnetwork",
-        "Disks": [
-                {
-                        "Name": "examplevmdisk",
-                        "SizeGB": 10,
-                        "Description": ""
-                }
-        ],
+        "Disks": [],
```
Comment (rawdaGastan) [inline grid-cli/docs/vm.md]: why disk is removed ?

File: grid-cli/docs/kubernetes.md [hunk 3]
```diff
@@ -37,9 +39,18 @@ Example:
...
-4:22PM INF deploying cluster
-4:22PM INF master planetary ip: 300:e9c4:9048:57cf:504f:c86c:9014:d02d
+11:43AM INF starting peer session=tf-1510734 twin=192
+11:43AM INF deploying network
+11:43AM INF deploying cluster
+11:43AM INF master wireguard ip: 10.20.2.2
+11:43AM INF master planetary ip: 300:e9c4:9048:57cf:d73d:eb4c:7a6d:503
+11:43AM INF master mycelium ip: 423:16f5:ca74:b600:ff0f:9ee:d57e:827a
+11:43AM INF worker1 wireguard ip: 10.20.2.3
+11:43AM INF worker0 wireguard ip: 10.20.2.3
```
Comment (rawdaGastan) [inline grid-cli/docs/kubernetes.md:49]: is it normal to have same ips for workers?

Comment (AbdelrahmanElawady) [reply grid-cli/docs/kubernetes.md:49]: nope not normal this is a bug

**Review by rawdaGastan — APPROVED** [review]

File: grid-cli/docs/kubernetes.md (hunk 3 above)
Comment (AbdelrahmanElawady) [reply grid-cli/docs/kubernetes.md:49]: created an issue for that #909

**Pull request merged**